.github/workflows/pull-request.yaml

```diff
@@ -12,8 +12,38 @@ on:
        CHARMHUB_TOKEN:
          required: false
 jobs:
+  ci-ignore:
+    name: Check against ignorelist
+    runs-on: ubuntu-latest
+    outputs:
+      files: ${{ steps.diff.outputs.files }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Zero means everything. We need `1` here, which is the default, so could be omitted entirely.